src/ogdf/fileformats/DotLexer.cpp

```diff
@@ -198,25 +198,34 @@ bool Lexer::tokenizeLine()
 }
 
 
-bool Lexer::match(const Token::Type &type)
+bool Lexer::match(const Token::Type &type, const bool word)
 {
-	return match(Token::toString(type));
+	return match(Token::toString(type), word);
 }
 
 
-bool Lexer::match(const std::string &str)
+bool Lexer::match(const std::string &str, const bool word)
 {
+	const size_t strLength = str.length();
```

Comment:
You do not need to introduce `strLength`. What's the benefit of it?

Reply:
It prevents repeated calls to `.length()`, which may be a minor optimization (although the compiler might do that already). It's also slightly easier to type.

Reply:
Hm, there shouldn't be a difference in the generated assembly code (except if you use `-O0`, probably)